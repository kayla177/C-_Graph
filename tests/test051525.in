ENTITY L6WV4JIE name_L6WV institution
ENTITY D06ZXA78 name_D06Z institution
ENTITY RFCNZ2UF name_RFCN author
ENTITY G7FW51JK name_G7FW journal
ENTITY QQ3Z7J6F name_QQ3Z paper
ENTITY YRV50UF6 name_YRV5 paper
ENTITY K8SIESHZ name_K8SI paper
ENTITY BP72WPPY name_BP72 institution
ENTITY VX7TP3OC name_VX7T conference
ENTITY AEKFKT7A name_AEKF conference
RELATIONSHIP G7FW51JK authored_by BP72WPPY 5.5
RELATIONSHIP D06ZXA78 cited_by AEKFKT7A 5.4
RELATIONSHIP G7FW51JK published_in QQ3Z7J6F 5.9
RELATIONSHIP K8SIESHZ affiliated_with L6WV4JIE 2.9
RELATIONSHIP BP72WPPY affiliated_with D06ZXA78 3.1
RELATIONSHIP RFCNZ2UF affiliated_with QQ3Z7J6F 1.2
RELATIONSHIP G7FW51JK collaborated_with VX7TP3OC 1.5
RELATIONSHIP AEKFKT7A authored_by VX7TP3OC 1.7
RELATIONSHIP VX7TP3OC published_in K8SIESHZ 1.2
RELATIONSHIP RFCNZ2UF presented_at AEKFKT7A 4.8
RELATIONSHIP BP72WPPY published_in K8SIESHZ 3.0
RELATIONSHIP G7FW51JK collaborated_with L6WV4JIE 6.5
RELATIONSHIP G7FW51JK published_in D06ZXA78 1.3
RELATIONSHIP D06ZXA78 authored_by VX7TP3OC 8.8
RELATIONSHIP RFCNZ2UF authored_by L6WV4JIE 1.1
PRINT RFCNZ2UF
PATH G7FW51JK BP72WPPY
PATH D06ZXA78 VX7TP3OC
HIGHEST
HIGHEST
PRINT QQ3Z7J6F
PRINT D06ZXA78
HIGHEST
HIGHEST
DELETE L6WV4JIE
EXIT
