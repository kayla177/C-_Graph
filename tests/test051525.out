success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
QQ3Z7J6F AEKFKT7A L6WV4JIE
G7FW51JK BP72WPPY 5.5
D06ZXA78 VX7TP3OC 8.8
L6WV4JIE VX7TP3OC 23.9
L6WV4JIE VX7TP3OC 23.9
G7FW51JK RFCNZ2UF
AEKFKT7A BP72WPPY G7FW51JK VX7TP3OC
L6WV4JIE VX7TP3OC 23.9
L6WV4JIE VX7TP3OC 23.9
success
